joblib/parallel.py

```diff
@@ -285,22 +285,24 @@ class Parallel(Logger):
 
         Parameters
         -----------
-        n_jobs: int, default: 1
-            The maximum number of concurrently running jobs, such as the number
-            of Python worker processes when backend="multiprocessing"
-            or the size of the thread-pool when backend="threading".
-            If -1 all CPUs are used. If 1 is given, no parallel computing code
-            is used at all, which is useful for debugging. For n_jobs below -1,
-            (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all
-            CPUs but one are used.
-        backend: str, ParallelBackendBase instance or None, \
-                default: 'multiprocessing'
-            Specify the parallelization backend implementation.
-            Supported backends are:
-
-            - "multiprocessing" used by default, can induce some
+        n_jobs: int, default: 1 The maximum number of concurrently running
+            jobs, such as the number of Python worker processes when
+            backend="loky" or the size of the thread-pool when
+            backend="threading". If -1 all CPUs are used. If 1 is given, no
+            parallel computing code is used at all, which is useful for
+            debugging. For n_jobs below -1, (n_cpus + 1 + n_jobs) are used.
+            Thus for n_jobs = -2, all CPUs but one are used. backend: str,
```

Comment:
formatting issue with `backend:str`

Reply:
And should be on the next line too (probably an effect from the automatic line wrapper, which fails with comment formatting)